rosdep/python.yaml

```diff
@@ -2164,6 +2164,19 @@ python-usb:
     raring: [python-usb]
     saucy: [python-usb]
     trusty: [python-usb]
+python-utm:
```

Comment:
The key should include `pip` to have an upgrade path once this package is being packaged by the platforms.